applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -930,6 +948,16 @@ int handle_at_pgps(enum at_cmd_type cmd_type)
 			if (interval < 10 || interval > 65535) {
 				return -EINVAL;
 			}
+#if defined(CONFIG_NRF_CLOUD_AGPS_FILTERED)
```

Comment:
No, P-GPS will also need to have scheduled downloads disabled even if AGPS_FILTERED is NOT defined.  So change this to `#if defined (CONFIG_NRF_CLOUD_PGPS)` .

The reason is that, in periodic tracking mode, the modem will try to download updated ephemeris data even though it was already given the P-GPS versions.  It is important to disable scheduled downloads in this case, so that the GNSS unit is not kept on for long periods unnecessarily.

Reply:
I think you should add Tommi Kangas as a reviewer -- he knows the scheduled download stuff the best.

Reply:
Ah, OK, maybe just remove this #if as CONFIG_NRF_CLOUD_PGPS is always enabled